code/game/objects/items/weapons/tools/_tools.dm

```diff
@@ -311,6 +311,8 @@
 //Cell reload
 /obj/item/weapon/tool/MouseDrop(over_object)
 	if((src.loc == usr) && istype(over_object, /obj/screen) && eject_item(cell, usr)) ///Formerly checked for /obj/screeninventory/hand. Not sure how that fits with bay's UI system
+		if (switched_on)
```

Comment:
this doesn't seem the right place to do this

Anything with a power cost should be consuming power each tick, and turning off if it fails to do so. Find out why that isn't working

Reply:
I've just checked few things and I've discovered that tools don't drain power passive. It drains power on use and when turned on.

Reply:
passive power drain is definitely a thing, it must be broken., fixing that is the way to resolve this problem